internal/bminventory/inventory_test.go

```diff
@@ -3322,6 +3325,74 @@ var _ = Describe("proxySettingsForIgnition", func() {
 	})
 })
 
+var _ = Describe("Register OCPCluster test", func() {
+
+	var (
+		bm             *bareMetalInventory
+		cfg            Config
+		db             *gorm.DB
+		ctx            = context.Background()
+		ctrl           *gomock.Controller
+		mockClusterAPI *cluster.MockAPI
+		mockHostApi    *host.MockAPI
+		mockS3Client   *s3wrapper.MockAPI
+		mockMetric     *metrics.MockAPI
+		mockK8sClient  *k8sclient.MockK8SClient
+	)
+
+	BeforeEach(func() {
+		Expect(envconfig.Process("test", &cfg)).ShouldNot(HaveOccurred())
+		ctrl = gomock.NewController(GinkgoT())
+		mockClusterAPI = cluster.NewMockAPI(ctrl)
+		mockHostApi = host.NewMockAPI(ctrl)
+		mockS3Client = s3wrapper.NewMockAPI(ctrl)
+		mockK8sClient = k8sclient.NewMockK8SClient(ctrl)
+		mockMetric = metrics.NewMockAPI(ctrl)
+		bm = NewBareMetalInventory(db, getTestLog(), mockHostApi, mockClusterAPI, cfg, nil, nil, mockS3Client, mockMetric, getTestAuthHandler(), mockK8sClient)
+	})
+
+	AfterEach(func() {
+		ctrl.Finish()
+	})
+
+	It("Register OCP cluster", func() {
+		configMap := v1.ConfigMap{}
+		configMap.Data = make(map[string]string)
```

Comment:
common content should be in before each or in a function

Reply:
fixed